server/fleet/activities.go

```diff
@@ -1598,3 +1599,92 @@ func LogRoleChangeActivities(
 	}
 	return nil
 }
+
+type ActivityEnabledVPP struct{}
+
+func (a ActivityEnabledVPP) ActivityName() string {
+	return "vpp_enabled"
+}
+
+func (a ActivityEnabledVPP) Documentation() (activity string, details string, detailsExample string) {
+	return "Generated when the VPP feature is enabled in Fleet.", "", ""
+}
+
+type ActivityDisabledVPP struct{}
+
+func (a ActivityDisabledVPP) ActivityName() string {
+	return "vpp_disabled"
+}
+
+func (a ActivityDisabledVPP) Documentation() (activity string, details string, detailsExample string) {
+	return "Generated when the VPP feature is disabled in Fleet.", "", ""
+}
+
+type ActivityAddedAppStoreApp struct {
+	SoftwareTitle string `json:"software_title"`
+	AppStoreID    string `json:"app_store_id"`
+	TeamName      string `json:"team_name"`
+}
+
+func (a ActivityAddedAppStoreApp) ActivityName() string {
+	return "added_app_store_app"
+}
+
+func (a ActivityAddedAppStoreApp) Documentation() (activity string, details string, detailsExample string) {
+	return "Generated when an App Store app is added to Fleet.", `This activity contains the following fields:
+- "software_title": Name of the App Store app.
+- "app_store_id": ID of the app on the Apple App Store.
+- "team_name": Name of the team to which this App Store app was added.`, `{
+  "software_title": "Logic Pro",
+  "app_store_id": "1234567",
+  "team_name": "Workstations"
+}`
+}
+
+type ActivityDeletedAppStoreApp struct {
+	SoftwareTitle string `json:"software_title"`
+	AppStoreID    string `json:"app_store_id"`
+	TeamName      string `json:"team_name"`
+}
+
+func (a ActivityDeletedAppStoreApp) ActivityName() string {
+	return "deleted_app_store_app"
+}
+
+func (a ActivityDeletedAppStoreApp) Documentation() (activity string, details string, detailsExample string) {
+	return "Generated when an App Store app is deleted from Fleet.", `This activity contains the following fields:
+- "software_title": Name of the App Store app.
+- "app_store_id": ID of the app on the Apple App Store.
+- "team_name": Name of the team from which this App Store app was deleted.`, `{
+  "software_title": "Logic Pro",
+  "app_store_id": "1234567",
+  "team_name": "Workstations"
+}`
+}
+
+type ActivityInstalledAppStoreApp struct {
+	HostID          int    `json:"host_id"`
+	HostDisplayName string `json:"host_display_name"`
+	SoftwareTitle   string `json:"software_title"`
+	AppStoreID      int    `json:"app_store_id"`
+	CommandUUID     string `json:"command_uuid"`
+}
+
+func (a ActivityInstalledAppStoreApp) ActivityName() string {
+	return "installed_app_store_app"
+}
+
+func (a ActivityInstalledAppStoreApp) Documentation() (string, string, string) {
```

Comment:
Looks like the docs should be re-generated, it doesn't seem to have all of those new activities.

Reply:
Ah I did generate, but I forgot to add the new ones to the array at the top 🤦